### PR TITLE
Refined the docs

### PR DESCRIPTION
**Pull request by ikoevska**
Note that I haven't touched the debug command.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -196,12 +197,15 @@ Run `tns help` to list all available commands. Run or `tns <Command> --help` to
...
+* `platform update <Platform>` updates the NativeScript runtime for the specified platform.
 * `prepare <Platform>` copies cross-platform and selected platform-specific content to the subdirectory for the target platform.
 * `build <Platform>` builds the project for the selected target platform.
 * `emulate <Platform>` builds the project for the selected target platform and runs it in the native emulator, if configured. 
 * `deploy <Platform> [--device <Device ID>]` deploys an already built application on connected device.
 * `run <Platform> [--device <Device ID>]` executes `prepare`, `build`, and `deploy`.
-* `list-devices` lists connected devices, including any running Android Virtual Devices.
+* `device` lists connected devices, including any running Android Virtual Devices or Genymotion virtual devices.
+* `device run` runs a selected application on a connected device.
+* `device list-applications` lists the installed applications on all connected devices.
```
Comment (Fatme) [inline README.md]: We should add `device log` command

Comment (ikoevska) [inline README.md]: Is it working for the NS CLI? I'm asking because it's not in the help.txt at all :)

Comment (Fatme) [inline README.md]: Yes, it works for NS CLI.
We missed it.

File: resources/help.txt [hunk 2]
```diff
@@ -189,53 +239,131 @@ Options:
...
-        tools
+--[/]--
+
+--[run|android]--
+Usage:
+    $ tns run android [--device <Device ID>] [--keyStorePath <File Path> --keyStorePassword <Password> --keyStoreAlias <Name> --keyStoreAliasPassword <Password>] [--release]
+    $ tns run android --emulator [<Emulator Options>] [--keyStorePath <File Path> --keyStorePassword <Password> --keyStoreAlias <Name> --keyStoreAliasPassword <Password>] [--release]
+    $ nativescript run android [--device <Device ID>] [--keyStorePath <File Path> --keyStorePassword <Password> --keyStoreAlias <Name> --keyStoreAliasPassword <Password>] [--release]
+    $ nativescript run android --emulator [<Emulator Options>] [--keyStorePath <File Path> --keyStorePassword <Password> --keyStoreAlias <Name> --keyStoreAliasPassword <Password>] [--release]
+
```
Comment (teobugslayer) [inline resources/help.txt]: run android also supports --geny, i think we should document it.

**Comment by teobugslayer** [comment]
Most excellent! :+1:

**Pull request merged**